src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -208,12 +215,19 @@ object EasyIngest {
       System.exit(13)
     }
 
+  private class CompositeException(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)} ${getStackTrace(t)}"))
+  private class CompositeExceptionMessages(throwables: List[Throwable]) extends RuntimeException(throwables.foldLeft("")((msg, t) => s"$msg\n${getMessage(t)}"))
   private implicit class ListTryExtensions[T](xs: List[Try[T]]) {
     def sequence: Try[List[T]] =
       if (xs.exists(_.isFailure))
         Failure(new CompositeException(xs.collect { case Failure(e) => e }))
       else
         Success(xs.map(_.get))
-  }
 
+    def squashedSequence: Try[List[T]] =
```

Comment:
I disagree with the first: when you have a signature like `def sequence(withStackTrace: Boolean): Try[List[T]]` you are much less flexible than with a signature like `def sequence(f: List[Throwable] => Throwable): Try[List[T]]` in which you can do all sorts of things. In the current use case you only have to give either the `CompositeExceptionMessage` or `CompositeException`, but I can imagine future work in which we want to do other stuff; filtering or something like that.

I'm fine with the second. [`sequence` originates from Haskell](http://hackage.haskell.org/package/base-4.9.0.0/docs/Prelude.html#v:sequence), in which it has the signature `sequence :: Monad m => t (m a) -> m (t a)`. Now that we propose to change the signature of this method in our code, it isn't a real `sequence` anymore. Something like `collectResults` makes more sense now.